commands/serverless-extra.go

```diff
@@ -84,34 +86,78 @@ the contents to the cloud incrementally as it detects changes.`,
 
 // RunServerlessExtraCreate supports the 'serverless init' command
 func RunServerlessExtraCreate(c *CmdConfig) error {
-	err := ensureOneArg(c)
+	if err := ensureOneArg(c); err != nil {
+		return err
+	}
+	project := c.Args[0]
+	overwrite, _ := c.Doit.GetBool(c.NS, "overwrite")
+	language, _ := c.Doit.GetString(c.NS, "language")
+
+	// Determine the kind and sample
+	kind, sample, ts, err := languageToKindAndSample(c, language)
 	if err != nil {
 		return err
 	}
-	output, err := RunServerlessExec(projectCreate, c, []string{flagOverwrite}, []string{flagLanguage})
+
+	// Make the config and various paths
+	config := configTemplate()
+	configFile := filepath.Join(project, "project.yml")
```

Comment:
Should we support  extension `yaml` instead ? [That is the recommended extension](https://yaml.org/faq.html). Not a blocker.

Reply:
Agreed we should to it.  I think not in this PR, though.